packages/ckeditor5-source-editing/docs/features/source-editing.md

```diff
@@ -0,0 +1,73 @@
+---
+category: features
+menu-title: Source editing
+modified_at: 2021-05-28
+---
+
+# Source editing
+
+The {@link module:source-editing/sourceediting~SourceEditing} feature provides the ability for viewing and editing the source of the document. The source editing plugin is a low-level document editing interface, while all the buttons and dropdowns located in a editor's toolbar are the high-level ones.
+
+The changes made to the document source will be applied to the editor's {@link framework/guides/architecture/editing-engine data model} only, if the editor understands (via loaded plugins) the given syntax. You will loose all changes done in the source editing mode unless they are compatible with what the initialized features understand. For example, if editor does not have a {@link features/horizontal-line horizontal line} plugin loaded, the `<hr>` tag added in the document source will be removed upon exit from the source editing mode.
```

Comment:
> You will loose all changes done in the source editing mode unless they are compatible with what the initialized features understand.

I'd rephrase it more to "You will lose all changes that editor features can't understand" or something similar.

Reply:
Done.